### PR TITLE
Update 20_install.adoc

### PR DESCRIPTION
**Pull request by apparatchick**
### All Submissions:

* [ ] Did you update the `changelog.adoc`?
* [x ] Does your PR affect the documentation?
* [x] If yes, did you update the documentation or create an issue for updating it? I have only updated the documentation.

The source of the documentation can be found in `/src/docs/`.

If you didn't find the time to update docs, please create an issue as reminder to do so.

### Your first submission

* [x] Welcome to the list of contributors! If you have any questions, feel free to ask them by creating a new issue
* [X ] Have you added your name to the list of [30_community.adoc](https://github.com/docToolchain/docToolchain/blob/ng/src/docs/10_about/30_community.adoc)?

inspiration: https://github.com/stevemao/github-issue-templates

### CODE REVIEW

File: src/docs/010_manual/20_install.adoc [hunk 1]
```diff
@@ -4,45 +4,55 @@
...
+`arch -x86_64 /bin/bash`
+----
+
+Now, type the following:
 
 [source, bash]
 ----
 cd <your project>
-curl -Lo dtcw doctoolchain.github.io/dtcw
+curl -Lo dtcw doctoolchain.github.io/dtcw of wget doctoolchain.github.io/dtcw
```
Comment (rdmueller) [inline src/docs/010_manual/20_install.adoc]: I am not sure about this change. Shouldn't this be a comment?

**Review by rdmueller — CHANGES_REQUESTED** [review]
thanx for updating the docs! Can you please take a look at my inline comment?

File: src/docs/010_manual/20_install.adoc (hunk 1 above)
Comment (apparatchick) [reply src/docs/010_manual/20_install.adoc]: Correct. I agree it should be kept as it is. As you can see, at line 45 is the wget.

Comment (rdmueller) [reply src/docs/010_manual/20_install.adoc]: btw: here is the preview of your changes:
https://deploy-preview-1021--dtc-docs-preview.netlify.app/010_manual/20_install.html

Comment (apparatchick) [reply src/docs/010_manual/20_install.adoc]: The "of wget octoolchain.github.io/dtcw" part can be removed.

**Review by apparatchick — COMMENTED** [review]
I've made the changes

**Review by rdmueller — APPROVED** [review]

**Comment by rdmueller** [comment]
thanx! I really appreaciate improvements to the docs :-D

**Pull request merged**